samples/java-spring-customer-registry-views-quickstart/src/main/java/customer/domain/Address.java

```diff
@@ -1,4 +1,4 @@
-package customer.api;
+package customer.domain;
```

Comment:
Moved to `customer.domain`. 

This is important because the page instruct the users to start from the Customer Registry quickstart. 

**Customer Registry** and **Customer Registry with Views** quickstart must be aligned.